http/handlers.go

```diff
@@ -121,10 +124,15 @@ func DecodeRequestWithHeaders(_ context.Context, req *http.Request, pathParams m
 }
 
 func getScheme(req *http.Request) string {
-	var xForwardedProtoHeader = req.Header.Get("X-Forwarded-Proto")
-
-	if xForwardedProtoHeader != "" {
-		return xForwardedProtoHeader
+	var forwardedHeader = req.Header.Get("Forwarded")
+
+	if forwardedHeader != "" {
+		// match should contain at least two elements if the protocol was specified in the Forwarded header.
+		//The first element will always be the 'proto=' capture, which we ignore.
```

Comment:
You are telling that `proto=` is the first element but the only sample provided in unit tests has proto as the second element.

Reply:
Ok... I confused it with the regex groups...